MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -335,15 +343,17 @@ - (void)detachAVPlayer {
 }
 
 - (void)monitorAVPlayerItem {
-    if (!_automaticVideoChange && !_didTriggerManualVideoChange) {
+    if ((!_automaticVideoChange && !_didTriggerManualVideoChange) || _isAdPlaying) {
         return;
     }
     if (_playerItem) {
         if (_didTriggerManualVideoChange) {
             _didTriggerManualVideoChange = false;
         }
         [self stopMonitoringAVPlayerItem];
+        
```

Comment:
extra ws

Reply:
I thought it provided better readability. Can remove if you disagree.